recipes/newrelic/infrastructure/ohi/kafka/rhel.yml

```diff
@@ -0,0 +1,57 @@
+# Visit our schema definition for additional information on this file format
```

Comment:
@NRhzhao could we may be combine debian/rhel in a single linux.yml recipe? I'm not seeing any implementation difference at this point

Reply:
@Julien4218  Sure, I wasn't sure because other recipes all have them separated.  I figure it's due to them having different package managers, so, during installation, we might keep them separated.